.copyrightignore

```diff
@@ -0,0 +1,39 @@
+# *******************************************************************************
+# Copyright (c) 2018, 2018 IBM Corp. and others
+# This program and the accompanying materials are made available under
+# the terms of the Eclipse Public License 2.0 which accompanies this
+# distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+# or the Apache License, Version 2.0 which accompanies this distribution and
+# is available at https://www.apache.org/licenses/LICENSE-2.0.
+#
+# This Source Code may also be made available under the following
+# Secondary Licenses when the conditions for such availability set
+# forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+# General Public License, version 2 with the GNU Classpath
+# Exception [1] and GNU General Public License, version 2 with the
+# OpenJDK Assembly Exception [2].
+#
+# [1] https://www.gnu.org/software/classpath/license.html
+# [2] http://openjdk.java.net/legal/assembly-exception.html
+#
+# SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+# *******************************************************************************/
+# *******************************************************************************
+#  THE FOLLOWING ARE EXAMPLES OF SYNTAXES copyrightCheck SUPPORTS:
+#  1.**/jobs/**# path contains "jobs" folder, but not root directory
+#  2./doc#root root level directory and file named "doc"
+#  3./debugtools/**/*.ignore# all .ignore file in the root directory debugtools
+#  4./*.ignore #all root directory .ignore files
+#  5.**/jobs/pipelines/#pipeline folder follows closely with jobs not in root directory
+#  6.*.ignore#all ignore files only
+#  7.**/jobs/**/pipelines/#directory path contains both jobs and pipelines not root directory
+#  8.*job* all things at contains characters in between stars
+# *******************************************************************************/
+**/jobs/**#comment
+/doc
+*job*
+#*.ignore
+/*.ignore
+/debugtools/**/*.ignore
+**/jobs/**/pipelines/
+**/jobs/pipelines/
```

Comment:
I think these were examples just for testing purposes ^ and should be removed.

Reply:
Not sure whether you wanted all of them removed but have done so. 
We can add others back if needed.